conformance_tests/sysman/test_sysman_device/src/test_sysman_device_helper.cpp

```diff
@@ -44,13 +87,32 @@ void get_ze_root_uuids(std::vector<ze_device_handle_t> ze_devices,
     }
     auto root_device_properties = lzt::get_device_properties(ze_root_device);
```

Comment:
Can we use get_ze_device_uuids here ?

Reply:
get_ze_device_uuids i am using to fetch all the device UUID's (used in flat mode). But in composite/combined mode root_device is obtanied first & uuid is retrieved from properties.  If we look at this angle it looks fine ?

Reply:
refactored to reduce code redundancy